mmv1/third_party/terraform/resources/resource_compute_instance_group_manager.go.erb

```diff
@@ -273,6 +273,13 @@ func resourceComputeInstanceGroupManager() *schema.Resource {
 				Default:     false,
 				Description: `Whether to wait for all instances to be created/updated before returning. Note that if this is set to true and the operation does not succeed, Terraform will continue trying until it times out.`,
 			},
+			"wait_for_instances_status": {
+				Type:        schema.TypeString,
+				Optional:    true,
+				Default:     "STABLE",
```

Comment:
I think we need a check on this value in Read- if it's `""`, set it to `"STABLE"`. That way we get away without a state upgrader and don't have a one-off diff for resources provisioned with older provider versions.